File: gateOne/day-One/java/TaskSix.java
import java.util.Scanner;
public class TaskSix{
	public static void main(String... args){

	Scanner input = new Scanner(System.in);
	int evenSum = 0;
	int scoreNumber = 10;
	int index = 1;
	for(int counter = 1; counter <= scoreNumber; counter++){
		int scoreCollector = input.nextInt();
		if(counter % 2 == 0){
			evenSum += scoreCollector;
			index++;
		}
	}

	double average = evenSum / index;
	
	System.out.print("the average of sum of even numbers is " + average);

	}




}

File: gateOne/day 1/python/tasksix.py
even_sum = 0
score_number = 10
index = 1
for count in range(score_number):
	score_collector = int(input("enter number: "))
	if count % 2 == 0:
		even_sum += score_collector
		index += 1


average = even_sum / index
print("the average is ", average)